packages/react-aria-components/src/Table.tsx

```diff
@@ -1219,3 +1223,34 @@ function RootDropIndicator() {
     </tr>
   );
 }
+
+// TOOD: no props for now, maybe get rid of this? Might be good to keep it just in case
+export interface TableLoaderProps {}
+
+export const UNSTABLE_TableLoader = createLeafComponent('loader', function TableLoader<T extends object>(props: TableLoaderProps, ref: ForwardedRef<HTMLTableRowElement>, item: Node<T>) {
```

Comment:
Now that this loader is only for load more, open to suggestions for the naming

Reply:
I also wondered if we wanted to go with one that's specific to the collection it's in or not. We are inconsistent at the moment. Items are specific because they accept different props per collection, but Section and DropIndicator are not (they pass a render function down via context). Which way do we want to go here?

Reply:
Does it accept different props? or do we think it will eventually?
does it matter if the loader is between items vs at the end? or in a different "level" even though we've flattened the collection?

Reply:
Oh I guess it does: TreeLoader has a `level` render prop. https://github.com/adobe/react-spectrum/pull/6494/files#diff-f6a41f61624976962cf916378a6e89a361a9c2067767dda15fbe6441890b996bR445

Reply:
ah thats right, forgot about that one. For now we will go with unique loader collection elements per collection component to match the pattern we have with items. DropIndicator makes sense to be shared since that is more of a useDnD hook think